estela-api/core/tasks.py

```diff
@@ -339,6 +433,19 @@ def record_job_coverage_event(job_id):
 
 
 def get_chain_to_process_usage_data(after_delete=False, project_id=None, job_id=None):
```

Comment:
Please document the **[caller](https://github.com/bitmakerla/estela/blob/main/estela-api/core/signals.py#L10)** too.

Reply:
Can you provide an example of list_of_process_functions?

Reply:
@joaquingx What's being done here is that the `list_of_process_functions` is populated automatically by looking into the Celery tasks in external apps with the name `process_usage_data`. This way, external apps can also process data from usage records when they are created. 

For example, [this Celery task named `process_usage_data`](https://github.com/bitmakerla/bitmaker-cloud-billing/blob/f0a051e7e4c590e819a0d33be4496dfba18dcd65/bitmaker_billing/tasks.py#L115) in one of our private modules that is installed as an externalApp:

Reply:
@joaquingx I've added documentation for the caller too. Please review it as well.

Reply:
Thanks!